test/functional/eval/reltime_spec.lua

```diff
@@ -0,0 +1,36 @@
+local helpers = require('test.functional.helpers')
+local clear, eq, ok = helpers.clear,  helpers.eq, helpers.ok
+local neq, execute, funcs  = helpers.neq, helpers.execute, helpers.funcs
+local reltime, reltimestr, reltimefloat = funcs.reltime, funcs.reltimestr, funcs.reltimefloat
+
+describe('Basic functionality of reltimestr and reltimefloat', function()
+  before_each(clear)
+
+  it('Checks', function()
+    local now = reltime()
+    execute('sleep 10m')
+    local later = reltime()
+    local elapsed = reltime(now)
+
+    neq(reltimestr(elapsed), '0.0')
+    print(reltimestr(elapsed))
+    ok((string.match(reltimestr(elapsed), "0%.0") and true) or false)
```

Comment:
Usually we test `~= nil`.

    ok(nil ~= string.match(reltimestr(elapsed), "0%.0"))